src/view/QuestionsList.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+//nu-mi trebe user aicea, ci in presenter si il pot trimite din smart component
+//da cand creez un nou question imi trebuie si useru, da pot sa-l adaug dupa
+//da-l in ma0sa, il harcodez
+
+import 'bulma/css/bulma.css';
+
+
+
+
+const QuestionsList = ({ questions, onCreateQuestion, searchTitle, searchTag, onChange }) => (
+
+            <div className={"column"}>
+                    <section className="hero is-fullheight is-dark is-bold">
+                            <div className="hero-body">
+                                    <div className="container">
+                                            <h1 className="title">
+                                                    Numberless questions
+                                            </h1>
+                                            <h2 className="subtitle">
+                                                    Reveal any mistery...scroll into the abyss
+                                            </h2>
+                                    </div>
+                            </div>
+                    </section>
+                    <section className="hero is-light is-bold">
+                            <div className={"columns"}>
```

Comment:
You don't need curly brackets if you just pass a constant (i.e. `<div className={"columns"}>` is the same as `<div className="columns">`)